tests/test_ble_client_edge_cases.py

```
@@ -317,3 +335,158 @@ def services(self) -> Any:
+    """unpair() should invoke backend unpair through _async_await."""
+    backend_calls: list[bool] = []
+    captured_timeout: list[float | None] = []
+
+    class _Backend:
+        async def unpair(self) -> None:
+            backend_calls.append(True)
+            return None
+
+    ble_client.bleak_client = cast(Any, _Backend())
+
+    def _run_awaitable(awaitable: Any, timeout: float | None = None) -> Any:
```

Ruff (0.15.4)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `awaitable`

(ANN401)